dist/src/main/resources/application.properties

```diff
@@ -61,3 +61,7 @@ camunda.identity.init.users[0].username=demo
 camunda.identity.init.users[0].password=demo
 spring.jpa.generate-ddl=false
 spring.jpa.hibernate.ddl-auto=validate
+
+# Multipart file uploads
+spring.servlet.multipart.max-file-size=4MB
+spring.servlet.multipart.max-request-size=4MB
```

Comment:
@tmetzke this two properties should have the same value of the `maxMessageSize` configured in zeebe, I think we should document this somewhere. Informing the customer to keep them aligned

Reply:
The default values are:

```
spring.servlet.multipart.max-file-size=1MB
spring.servlet.multipart.max-request-size=10MB
```

Reply:
Hmm, interesting. Is this configuring the server to allow those sizes?

Reply:
Yes, but they still need to be the same value configured in zeebe for the maxMessageSize. Otherwise we will face this [error](https://github.com/camunda/camunda/blob/4d9013e77cdc01678e59a66c9643492d0a0d6e9d/zeebe/broker/src/main/java/io/camunda/zeebe/broker/transport/commandapi/CommandApiRequestHandler.java#L151-L154)

Reply:
But only if users put up a payload that is too big, right? If that is the case, it's totally fine that the defaults of the servlet stay untouched, I think. The message size is the limitation inside the broker and if it cannot handle the request, it will be rejected. The REST request should fail with a reasonable error, explaining that the payload was too big.

Is there any behavior that is unexpected in that regard so we should change those properties?

Reply:
If the customers use a lower `maxMessageSize` (our default is 4MB) there is no problem, cause the request will be rejected by the broker. 
But customers can use also an higher `maxMessageSize` (> 4MB), in this case the request will be rejected by Spring itself

Reply:
Yeah, I see what you mean. OK, let's adjust this here and think about proper documentation and configuration downstream in a follow-up issue 👍